qa/docker/buildbot.cfg

```diff
@@ -126,11 +125,13 @@ pcaps_list = [ os.path.join(PCAP_PATH, pcap) for pcap in pcaps_list if pcap.ends
 factory_stress_pcap = SuriBuildFactory()
 # run the tests (note that this will require that 'trial' is installed)
 factory_stress_pcap.addStep(ShellCommand(command=["./autogen.sh"]))
-factory_stress_pcap.addStep(ShellCommand(command=["./configure","--enable-debug-validation"],env={"CFLAGS" : "-fsanitize=address -fno-omit-frame-pointer"}))
+factory_stress_pcap.addStep(ShellCommand(command=["./configure","--enable-debug-validation", "--enable-rust","ac_cv_func_malloc_0_nonnull=yes","ac_cv_func_realloc_0_nonnull=yes"],env={"CFLAGS" : "-fsanitize=address -fno-omit-frame-pointer"}))
 factory_stress_pcap.addStep(ShellCommand(command=["make"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "make","install"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "rm", "-f", "/usr/local/etc/suricata/suricata.yaml"]))
 factory_stress_pcap.addStep(ShellCommand(command=["sudo", "make","install-conf"]))
+if os.path.exists('/data/oisf/qa/suricata-verify/run.py'):
+    factory_stress_pcap.addStep(ShellCommand(command=["/data/oisf/qa/suricata-verify/run.py","--outdir","/tmp"]))
 factory_stress_pcap.addStep(ShellCommand(command=["make","clean"]))
```

Comment:
this one then also needs to be moved otherwise the src/suricata is gone again

Reply:
Tests seem to show it is ok. `make clean` is after the test. I'm rebasing and resubmitting.